package.json

```diff
@@ -44,6 +44,7 @@
     "mime": "2.3.1",
     "minimatch": "3.0.4",
     "mkdirp": "0.5.1",
+    "mv": "^2.1.1",
```

Comment:
Should it not already installed by Logger  ? https://github.com/trentm/node-bunyan/blob/fe31b83e42d9c7f784e83fdcc528a7c76e0dacae/package.json#L27

Reply:
It's optional and this forces it to be there. Their code does if(mv): https://github.com/trentm/node-bunyan/blob/master/lib/bunyan.js#L96

Reply:
Remove the `^` please